tensorflow/tools/api/golden/v1/tensorflow.pbtxt

```diff
@@ -1900,6 +1900,14 @@ tf_module {
     name: "rint"
     argspec: "args=[\'x\', \'name\'], varargs=None, keywords=None, defaults=[\'None\'], "
   }
+  member_method {
+    name: "roi_align"
```

Comment:
This probably belongs in tf.image

Reply:
Like in #28745 I created this file with  `bazel-bin/tensorflow/core/api_def/update_api_def`. How can I make it part of tf.image?

Reply:
Done